application/helpers/admin/import_helper.php

```diff
@@ -2357,13 +2357,16 @@ function XMLImportSurvey($sFullFilePath, $sXMLdata = null, $sNewSurveyName = nul
             if (!isset($insertdata['quotals_quota_id']) || (int)$insertdata['quotals_quota_id'] < 1) {
                 continue;
             }
-            $insertdata['autoload_url'] = 0; // used to bypass urlValidator check in QuotaLanguageSetting model
             $insertdata['quotals_quota_id'] = $aQuotaReplacements[(int) $insertdata['quotals_quota_id']]; // remap the qid
             unset($insertdata['quotals_id']);
-            $quotaLanguagesSetting = new QuotaLanguageSetting();
+            $quotaLanguagesSetting = new QuotaLanguageSetting('import');
```

Comment:
I tried it without the "import" scenario. That means the following:
The survey will be imported and the admin importing the survey gets a hint, that he should add the urls for the other languages. As we do not allow (try to add a quota with autoload_url=on and 3 languages, add only url for the first language) to save the quota, without all urls being set, it makes no sense to allow this when importing a survey.
No data will be lost during import.

Reply:
>  No data will be lost during import.

False : QuotaLanguageSettings was not saved : then text are lost